testutil/cli/exec.go

```diff
@@ -9,11 +9,12 @@ import (
 	sdkcli "github.com/cosmos/cosmos-sdk/testutil/cli"
 	"github.com/cosmos/cosmos-sdk/testutil/network"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
 	"github.com/provenance-io/provenance/testutil/assertions"
 	"github.com/provenance-io/provenance/testutil/queries"
 )
 
-// CLITxExecutor helps facilitate the execution and testing of a CLI command.
+// TxExecutor helps facilitate the execution and testing of a CLI command.
```

Comment:
This should stay CLITxExecutor because it's specific only to CLI stuff. E.g. it shouldn't be used for sims or keeper stuff.

Reply:
Actually. Nevermind. I forgot that I put it in the `cli` package. So it's a `cli.TxExecutor`. All good. Sorry.